packages/shared/src/utils/env.ts

```diff
@@ -8,6 +8,9 @@ export enum Env {
   AwsRegion = 'AWS_REGION',
   SqsWorkerQueueUrl = 'SQS_WORKER_QUEUE_URL',
 
+  // GitHub
+  GitHubAccessToken = 'GITHUB_ACCESS_TOKEN',
```

Comment:
It is probably worth adding this variable to the `checkRequiredEnvironmentVariables` payload in the `worker/src/index.ts` file

Reply:
Up to you, but I prefer to be sure that the worker application works with any task if it is successfully deployed